src/meta/client/MetaClient.cpp

```diff
@@ -980,9 +982,20 @@ folly::Future<StatusOr<bool>> MetaClient::heartbeat() {
     thriftHost.set_ip(localHost_.first);
     thriftHost.set_port(localHost_.second);
     req.set_host(std::move(thriftHost));
+    if (clusterMan_) {
+        req.set_clusterId(clusterMan_->getClusterId());
+    } else {
+        req.set_clusterId(0);
+    }
     return getResponse(std::move(req), [] (auto client, auto request) {
         return client->future_heartBeat(request);
-    }, [] (cpp2::HBResp&& resp) -> bool {
+    }, [this] (cpp2::HBResp&& resp) -> bool {
+        if (resp.code == cpp2::ErrorCode::SUCCEEDED) {
+            if (clusterMan_ && !clusterMan_->isClusterIdSet()) {
```

Comment:
is it really need do this check? because the service can not start if the clusterMan_ init failed.

Reply:
Yes, fixed it!